pkgs/development/compilers/rust/rustc.nix

```diff
@@ -97,11 +97,11 @@ stdenv.mkDerivation {
     rm -vr src/test/run-make/linker-output-non-utf8 || true
 
     # Remove test targeted at LLVM 3.9 - https://github.com/rust-lang/rust/issues/36835
-    rm -vr src/test/run-pass/issue-36023.rs || true
+    rm -fvr src/test/run-pass/issue-36023.rs
```

Comment:
feel free to revert these to lines but either way it is still a hack since we are using rustc.nix with 3 different revisions of rust.